docs/glossary.md

```diff
@@ -45,7 +45,7 @@ Use the alphabatized list below to find the answer to your single-term questions
 
 ## D
 
-- [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.
+- [**Dashboards**](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md): Out-of-the box visual presentation of metrics that allows you to make sense of your infrastructure and its health and performance.
```

Comment:
Should this be **Dashboard**? Why plural?

Reply:
yeah, I'll do it singular, nice catch